README.md

```diff
@@ -21,17 +21,69 @@ dependencies, you need to require
 }
 ```
 
-Please, read the website to [get more informations about how to
-install](http://hoa-project.net/Source.html).
+And add `RulerBundle` to your `AppKernel`:
+
+```php
+// app/AppKernel.php
+
+class AppKernel extends Kernel
+{
+    // …
+
+    public function registerBundles()
+    {
+        $bundles = array(
+            // …
+            new Hoathis\Bundle\RulerBundle\HoathisRulerBundle(),
+        );
+
+        // …
+    }
+}
+```
 
 ## Quick usage
 
-…
+### Ruler service
 
-## Documentation
+The `RulerBundle` defines a `hoathis.ruler` service that can be used as
+described in [Ruler's documentation](http://hoa-project.net/En/Literature/Hack/Ruler.html).
+
+```php
+public function indexAction(Request $request)
+{
+    // 1. Write a rule.
+    $ruler = $this->get('hoathis.ruler');
+    $rule  = 'group in ["customer", "guest"] and points > 30';
 
-Different documentations can be found on the website:
-[http://hoa-project.net/](http://hoa-project.net/).
+    // 2. Create a context.
+    $context           = new \Hoa\Ruler\Context();
```

Comment:
I think it makes sense because the ruler really is a service: the same instance can be used each time we want to make an assertion and the ruler object provides this service.
On the other hand, we have to build different `Hoa\Ruler\Context` instances to represent the data on which the assertions will be made and that's not something we should put in the container (I think).